pkg/watch/health.go

```diff
@@ -36,6 +36,7 @@ const TTL = 60 * time.Second
 // has a running MonitorHealth go routine
 type PodWatch struct {
 	manifest pods.Manifest
+	client   *http.Client
```

Comment:
(I thought this was in a function, not in a struct, so was confused. Upon further review I could not have been correct because if it were in a function it needs to be `var client *http.Client`)

I think what you have written here is preferable. I agree with you that adding methods to `Store` for checking seems a bit weird.